src/main/java/seedu/duke/Constants.java

```diff
@@ -72,11 +72,14 @@
     public static final String COMMAND_CLASS_PREFIX = "seedu.duke.command.";
 
     // Delimiters for storage class
-    public static final String ID_DELIMITER = "||";
+    public static final String ID_DELIMITER = "~~";
     public static final String DATE_DELIMITER = "``";
-    public static final String SYMPTOM_DELIMITER = "^^";
+    public static final String SYMPTOM_DELIMITER = "%%";
     public static final String DIAGNOSIS_DELIMITER = "##";
     public static final String PRESCRIPTION_DELIMITER = "@@";
-    public static final String RECORDS_DELIMITER = "[R]";
+    public static final String RECORDS_DELIMITER = "!R!";
+    public static final String DETAILS_DELIMITER = "\t";
```

Comment:
I would suggest use another delimiter as "\t" is a white space character. I am not sure whether it will cause unexpected behavior since I need to filter all these delimiter in Parser as well.